src/App.js

```diff
@@ -4,13 +4,13 @@ import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
+    <div className="app">
```

Comment:
did you spin up the app to test if the CSS works? I figured if you update the CSS in the file, you would have to update the names in `App.css` too. 

It's a good practice to test your code before you resubmit for review.

Reply:
Forgot that. Got it.